app/models/status.rb

```diff
@@ -256,7 +256,7 @@ def non_sensitive_with_media?
   end
 
   def reported?
-    @reported ||= Report.where(target_account: account).unresolved.where('? = ANY(status_ids)', id).exists?
+    @reported ||= Report.where(target_account: account).where('? = ANY(status_ids)', id).exists?
```

Comment:
Let's investigate this code addition.

Reply:
actually it was a code removal that I did (there have been no changes to upstream on this line).

It's needed in order to access private statuses and direct messages from Cinder.

I made a comment about it here in my PR on upstream: [link](https://github.com/mastodon/mastodon/pull/27675/files#r1382155905)

We could add some comment about it directly in the code here if that would be helpful for future